.rubocop_fixme.yml

```diff
@@ -47,6 +47,10 @@ Metrics/MethodLength:
     - 'lib/hyrax/rails/routes.rb'
     - 'spec/support/**/*'
 
+Style/FormatStringToken:
```

Comment:
rubocop says
```
== config/routes.rb ==
C: 10: 58: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
```
But that doesn't seem to work, at least in the routes definitions.